packages/react/src/components/DateTimePicker/DateTimePicker.test.jsx

```diff
@@ -685,6 +694,86 @@ describe('DateTimePicker', () => {
     expect(applyBytton).toBeDisabled();
   });
 
+  it('should disable apply button when absolute TimePickerSpinner input is invalid on the same day', () => {
+    const defaultValues = {
+      timeRangeKind: PICKER_KINDS.ABSOLUTE,
+      timeRangeValue: {
+        startDate: '2020-04-01',
+        startTime: '12:34',
+        endDate: '2020-04-01',
+        endTime: '11:49',
+      },
+    };
+    const wrapper = mount(<DateTimePicker {...dateTimePickerProps} defaultValue={defaultValues} />);
```

Comment:
Looking great. Thanks, @eosineye1. I should have mentioned this yesterday, but didn't think of it. Can you switch these tests over to using testing-library instead of enzyme. We're working on replacing all the existing enzyme tests and write all new tests in testing library.